pkg/registry/apis/provisioning/register.go

```diff
@@ -504,7 +508,7 @@ func (b *APIBuilder) GetPostStartHooks() (map[string]genericapiserver.PostStartH
 			}
 
 			// Informer with resync interval used for health check and reconciliation
-			sharedInformerFactory := informers.NewSharedInformerFactory(c, 10*time.Second)
+			sharedInformerFactory := informers.NewSharedInformerFactory(c, 60*time.Second)
```

Comment:
bumping this up gets my system working a bit better -- otherwise it is constantly checking webhook status for a system I know is not trying to run webooks